### PR TITLE
[wiz] Enable addon finder support

### LINKED ISSUE #17847: [wiz] Support suggested addons
The WiZ discovery packet requires IP address and MAC address, but the latter needs to be added to IpAddonFinder in core first.

### PR DESCRIPTION
**Pull request by ccutrer**
Closes #17847

### CODE REVIEW

**Comment by ccutrer** [comment]
Depends on https://github.com/openhab/openhab-core/pull/4477

File: bundles/org.openhab.binding.wiz/src/main/resources/OH-INF/addon/addon.xml [hunk 1]
```diff
@@ -8,4 +8,38 @@
...
+				</discovery-parameter>
+				<discovery-parameter>
+					<name>destPort</name>
+					<value>38899</value>
+				</discovery-parameter>
+				<discovery-parameter>
+					<name>requestPlain</name>
+					<value>{"method":"registration","id":1,"params":{"phoneIp":"$srcIp","register":false,"phoneMac":"$srcMac"}}</value>
+				</discovery-parameter>
+				<discovery-parameter>
```
Comment (andrewfg) [inline bundles/org.openhab.binding.wiz/src/main/resources/OH-INF/addon/addon.xml:27]: I think you need to run spotless..

**Review by andrewfg — APPROVED** [review]
Otherwise LGTM

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**